.vscode/mcp.json

```diff
@@ -1,5 +1,9 @@
 {
   "servers": {
+    "atlassian": {
+      "type": "http",
```

Comment:
The URL looks like the type should be "sse", but the type is "http" with the same URL when installing this MCP server via the Extensions menu.